src/form.css

```diff
@@ -167,6 +168,9 @@
   transition-duration: 0s;
   opacity: 1;
 }
+[data-unfeed] :not([data-unfeed-type]) .unfeed__icon-button {
```

Comment:
Hide the close button when the widget is shown in container and no type is selected.

Left: widget in container, right: widget in toggle.

![image](https://user-images.githubusercontent.com/7823011/202867507-cb11f820-6a90-4368-8c13-6c7a75947217.png)